src/addons/addonmessage.cpp

```diff
@@ -224,3 +227,23 @@ qint64 AddonMessage::planDateRetranslationInternal(
 
   return 86400 + secsTo;
 }
+
+void AddonMessage::enable() {
+  Addon::enable();
+
+  maybeNotify();
+}
+
+void AddonMessage::maybeNotify() {
+  NotificationHandler* notificationHandler = NotificationHandler::instance();
+  if (!notificationHandler) {
+    // When loading at startup the notification handler is not available.
+    // That is fine, we no not want to notify when loading at startup.
+    return;
+  }
+
+  if (enabled() && !m_isRead) {
+    NotificationHandler::instance()->newInAppMessageNotification(m_title.get(),
+                                                                 m_title.get());
```

Comment:
This means that we notify at any startup right?

Reply:
No, because the NotificationHandler is not available at startup and we check for that right before that check. That is fragile though, I'll see if I can figure out something better.